README.md

```diff
@@ -2,3 +2,23 @@ Drupal Training
 2222
 ===============
 Some changes
+24.12
```

Comment:
Remove Russian content of README.md and add couple of sentences in English describing the purpose of the module